src/helpfile

```diff
@@ -12,7 +12,9 @@ Options:
 -c 2            Set coords (1, 2, 3 dimensions)
 -D 0.0          Set's offset for kill (-K flag) distance radially
 -d data         Set data directory - where to store / read data (here in data/)
--E              Perform energy calculation every writing step
+-E 0.0001 123   Perform energy calculations, exiting early if the change in energy is sufficiently small.
```

Comment:
Would we be able to keep these lines within 80 columns? It doesn't matter much for the actual code, but the help menu will be output to a terminal, which is often 24x80 by default